solutions/cf-1000C.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: cf-1000C
+source: CF
+title: Covered Points COunt
+author: Jesse Choe
+---
+
+[Official Editorial](http://codeforces.com/blog/entry/60288)
+
+
+# Solution (Coordinate Compression)
+
+Due to the large constraints on $l_i$ and $r_i$, it is impossible to apply brute force the number of intersecting segments for each point on the coordinate line. Nonetheless, we can compress the coordinates and "pretend" as if the coordinates were between $1$ and $n$.
+
+This allows us to apply prefix sums on each point in our compressed "number line" and retransform these compressed coordinates to the original endpoints $l_i$ and $r_i$. To compress the coordinates, we can either sort the coordinates or use a `HashMap`.
```

Comment:
just hash map, `HashMap` is java-specific

Reply:
also why is number line in quotes

Reply:
^^ just say map